packages/nextra-theme-docs/src/css/styles.css

```diff
@@ -790,9 +743,6 @@ table tr td {
   code {
     direction: ltr;
   }
-  .anchor-icon {
-    @apply ml-0 mr-2 inline-block;
-  }
```

Comment:
added `px-2` instead and in rtl it's looks fine
![image](https://user-images.githubusercontent.com/7361780/182024287-367d31c5-e0a5-4a8e-b726-4a3fb88af168.png)